content/sensu-go/5.19/reference/backend.md

```diff
@@ -1263,7 +1269,15 @@ Here are some log rotate sample configurations:
 }
 {{< /highlight >}}
 
-#### sysvinit
+Without the `postrotate` script, the backend will not reload.
+This will cause sensu-backend (and sensu-agent, if translated for the Sensu agent) to no longer write to the log file, even if logrotate recreates the log file.
```

Comment:
```suggestion
This will cause sensu-backend (and sensu-agent, if translated for the Sensu agent) to no longer write to the log file, even if logrotate recreates the log file.

In this script, `systemctl reload` sends a _SIGHUP_ signal to the sensu-backend process.

- In Sensu Go versions 5.19.2 and earlier, this reload causes sensu-backend to fully restart.
If you are running a clustered backend, all cluster members will restart at the same time.
- In [Sensu Go version 5.19.3][30] and later, the _SIGHUP_ signal causes only event logging components to reload, so cluster members will not simultaneously restart.
```